### PR TITLE
replacing BIB_BLUE_BUTTON_BASE_URL to BIG_BLUE_BUTTON_BASE_URL

### PR DESCRIPTION
**Pull request by sumanvpacewisdom**

### CODE REVIEW

File: src/envVariables.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ let enviromentVariables = {
 		optional: process.env.ENABLE_EMAIL_FOR_REPORT_ISSUE === 'true' ? false : true,
 		default: 'user_issue_reported',
 	},
-	BIG_BLUE_BUTTON_BASE_URL: {
+	BIB_BLUE_BUTTON_BASE_URL: {
```
Comment (rakeshSgr) [inline src/envVariables.js:140]: @suman Can you update the code BIG_BLUE_BUTTON_BASE_URL, so it will be proper

Comment (sumanvpacewisdom) [reply src/envVariables.js:140]: then no need of merging this pr , its already having the same @rakeshSgr

Comment (rakeshSgr) [reply src/envVariables.js:140]: but in code we still refer this BIB_BLUE_BUTTON_BASE_URL

Comment (rakeshSgr) [reply src/envVariables.js:140]: update the code where ever we use this key

Comment (sumanvpacewisdom) [reply src/envVariables.js:140]: @rakeshSgr ok

File: src/api-doc/mentoring-2.5-setup.md [hunk 2]
```diff
@@ -366,7 +366,7 @@ USER_SERVICE_BASE_URL='/user/'
 BIG_BLUE_BUTTON_URL=https://dev.some.temp.org
 
 # Big blue button base URL
-BIB_BLUE_BUTTON_BASE_URL=/bigbluebutton/
+BIG_BLUE_BUTTON_BASE_URL=/bigbluebutton/
```
Comment (rakeshSgr) [inline src/api-doc/mentoring-2.5-setup.md:369]: @sumanvpacewisdom for older version of docs, we should't make any env changes, the docker images dependent on those env keys only

**Pull request closed without merging**